internal/host/host.go

```diff
@@ -513,9 +578,50 @@ func (m *Manager) SetUploadLogsAt(ctx context.Context, h *models.Host, db *gorm.
 	return nil
 }
 
+func canonizeConnectivity(connectivityReportStr string) string {
+	var connectivityReport models.ConnectivityReport
+	err := json.Unmarshal([]byte(connectivityReportStr), &connectivityReport)
+	if err != nil {
+		return connectivityReportStr
+	}
+	for _, h := range connectivityReport.RemoteHosts {
+		for _, l3 := range h.L3Connectivity {
+			l3.AverageRTTMs = 0
+			l3.PacketLossPercentage = 0
+		}
+		l3 := h.L3Connectivity
+		sort.Slice(l3, func(i, j int) bool {
```

Comment:
maybe sorting should be done in agent?

Reply:
Maybe, but we need to verify here that it is correct.  I wouldn't remove from here.

Reply:
we must have sort in agent to make it's cache to work well

Reply:
maybe we can have it as pkg in service and use this func in agent?

Reply:
Since we have rtts in the connectivity reply, I think that the cache will usually not give much benefit for this step-type.  If it will, then this reply will not be received in the service.
Anyway, sorting should be done on agent level too.